syft/execution/role.py

```diff
@@ -47,6 +49,8 @@ def __init__(
         self.state = state or State()
         self.tracing = False
 
+        self.torch = FrameworkWrapper(package=self.worker.torch, role=self, owner=self.worker)
```

Comment:
I think we could even have a for loop to add all the frameworks we support here

Reply:
Oh yeah! That would be cool.